src/server/services/routes/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const express = require('express');
+const router = express.Router();
+var accountController;
+
+router.connect = function() {
+    accountController = require('../controllers/accountController.js');
+
+    router.get('/', function(req, res) {
+        res.json({ message: 'Welcome to the api' });
+    });
+
+    router.get("/account", accountController.getAllAccounts);
```

Comment:
Excellent! It is actually really rare, even at large companies, to see the HTTP methods used correctly. Bravo! 😺🎉